### PR TITLE
Fix find similar incidents

### PR DESCRIPTION
**Pull request by erezh31**
## Status
Ready

## Related Issues
fixes https://github.com/demisto/etc/issues/19174

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 please review changelog

File: Scripts/FindSimilarIncidents/FindSimilarIncidents.py [hunk 1]
```diff
@@ -257,90 +243,119 @@ def merge_incident_fields(incident):
...
+                'Type': entryTypes['note'],
+                'Contents': markdown_result,
+                'EntryContext': context}
+    else:
+        did_not_found_duplicates()
+
+
+if __name__ in ['__main__', '__builtin__', 'builtins']:
+    entry = main()
+    demisto.results(entry)
```
Comment (Itay4) [inline Scripts/FindSimilarIncidents/FindSimilarIncidents.py]: do it in the main function, here just call `main`

File: Scripts/FindSimilarIncidents/FindSimilarIncidents.py [hunk 2]
```diff
@@ -221,6 +200,8 @@ def is_text_equal_by_x_different_words(text1, text2, number_of_different_words,
 
 
 def verify_map_equals(values_map1, values_map2, equality_map):
+    if not equality_map or len(equality_map) == 0:
```
Comment (Itay4) [inline Scripts/FindSimilarIncidents/FindSimilarIncidents.py]:
```suggestion
    if not equality_map:
```

**Comment by kirbles19** [comment]
@erezh31 Done.

**Pull request merged**